q.go

```diff
@@ -0,0 +1,230 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+)
+
+var counter = 81
+var a [9][9]byte
+var colors = [7]byte{'a', 'b', 'c', 'd', 'e', 'f', 'g'}
```

Comment:
Let's make these colors as enums. http://stackoverflow.com/questions/14426366/what-is-an-idiomatic-way-of-representing-enums-in-go